src/app/components/modal/modal.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, Output, EventEmitter, OnInit } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { RestApiService } from 'src/app/services/restapi.service';
+import { OpenBoardService } from 'src/app/services/open-board.service';
+import { SwalService } from 'src/app/services/swal.service';
+
+@Component({
+  selector: 'app-modal',
+  templateUrl: './modal.component.html',
+  styleUrls: ['./modal.component.css']
+})
+export class ModalComponent implements OnInit {
+  title!: string;
+  description!: string;
+  type!: string;
+
+  @Output() closeModalEvent = new EventEmitter();
```

Comment:
Please try to follow a single approach to the declaration sequence. I can propose you the following

@decorated properties
public properties
private properties
constructor
lifecycle hooks
public methods
private methods